### PR TITLE
feat: adds dropdown component

### PR DESCRIPTION
**Pull request by OlegSymonchuk**

### CODE REVIEW

File: components/common/Dropdown/Dropdown.interfaces.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export interface SelectorItem {
```
Comment (kucherenko) [inline components/common/Dropdown/Dropdown.interfaces.tsx]: interfaces should not be in tsx files, use ts files for that

File: styles/globalStyle.ts [hunk 2]
```diff
@@ -9,7 +9,7 @@ const COLORS = {
   deepSea: '#064aff',
   deepSea80: 'rgba(6, 135, 255, 0.8)',
   graphiteGrey: '#333333',
-  crystalWhite: '#ffffff',
+  crystalWhite: 'rgba(255, 255, 255)',
```
Comment (kucherenko) [inline styles/globalStyle.ts:12]: colors should be in one style rgba(... of #fff...

**Review by kucherenko — CHANGES_REQUESTED** [review]

File: components/common/Dropdown/Dropdown.interfaces.tsx (hunk 1 above)
Comment (OlegSymonchuk) [reply components/common/Dropdown/Dropdown.interfaces.tsx]: Renamed

File: styles/globalStyle.ts (hunk 2 above)
Comment (OlegSymonchuk) [reply styles/globalStyle.ts:12]: Converted

**Review by kucherenko — APPROVED** [review]

**Pull request merged**